src/modules/fhir/observation.ts

```diff
@@ -0,0 +1,62 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { z } from 'zod'
+
+export const FHIR_OBSERVATION_RESOURCE_TYPE = 'Observation'
```

Comment:
Would it make sense to keep enum of FHIR resource types, instead of multiple constants?